Dockerfile

```diff
@@ -158,44 +178,46 @@ RUN rm /etc/apache2/sites-enabled/000-default.conf
 COPY --from=builder /tmp/local/ /opt/perl/local/
 ENV PERL5LIB="/opt/product-opener/lib/:/opt/perl/local/lib/perl5/"
 ENV PATH="/opt/perl/local/bin:${PATH}"
+# Create writable dirs and change ownership to www-data
+RUN \
+    mkdir -p var/run/apache2/ && \
+    chown www-data:www-data var/run/apache2/ && \
+    for path in users products product_images orgs new_images logs; do \
```

Comment:
I think we need to add "tmp" here, as PO tries to create /mnt/podata/tmp/templates

Otherwise I get this error:
```

# Run build_lang.pl
docker-compose --env-file=.env run --rm backend ls -lrt /mnt/
Starting po_memcached_1 ... done
Starting po_postgres_1  ... done
total 4
drwxr-xr-x 10 root root 4096 Nov  5 15:20 podata
docker-compose --env-file=.env run --rm backend perl -I/opt/product-opener/lib -I/opt/perl/local/lib/perl5 /opt/product-opener/scripts/build_lang.pl
Starting po_memcached_1 ... done
Starting po_postgres_1  ... done
Status: 500
Content-type: text/html

<h1>Software error:</h1>
<pre>mkdir /mnt/podata/tmp: Permission denied at /opt/perl/local/lib/perl5/x86_64-linux-gnu-thread-multi/Template/Provider.pm line 378.

```

Reply:
done.